_chapters/chapter_01.md

```diff
@@ -9,22 +9,30 @@ layout: rails_tutorial
 {% endsectionheader %}
 
 {% protip %}
-  Throughout the tutorial, you're going to run a lot of Terminal commands. For example, we might ask you to do something like
+  Throughout the tutorial, you're going to run a lot of commands on the command line. For example, we might ask you to do something like
 
-  $ Run `pwd` to print your current working directory.
+  > Run `$pwd` to print your current working directory.
```

Comment:
`$pwd` looks good in the browser, but it might be confusing. You're going to type `pwd` in the Terminal, not `$pwd`.